sources/osg/Uniform.js

```diff
@@ -54,15 +54,25 @@ Uniform.prototype = {
 
     apply: function UniformApply( gl, location ) {
 
-        if ( !this._cache )
+        if ( !this._cache || this._cacheDirty === true ) {
             this._cache = gl[ this._glCall ];
-
+            this._dirtyCache = false;
```

Comment:
this._cacheDirty = false;

Also why exposing gl call signature? I think it's supposed to be kept private